src/Blocks/BlockHandler.h

```diff
@@ -111,15 +111,7 @@ class cBlockHandler
 	/** Checks if the player can build "inside" this block.
 	For example blocks placed "on" snow will be placed at the same position. So: Snow ignores Build collision
 	*/
-	virtual bool DoesIgnoreBuildCollision(void);
-
-	/** Similar to DoesIgnoreBuildCollision(void), but is used for cases where block's meta or
-	player's item-in-hand is needed to determine collision (thin snow) */
-	virtual bool DoesIgnoreBuildCollision(cPlayer *, NIBBLETYPE a_Meta)
-	{
-		UNUSED(a_Meta);
-		return DoesIgnoreBuildCollision();
-	}
+	virtual bool DoesIgnoreBuildCollision(cChunkInterface & ChunkInterface, Vector3i a_RelPos, cPlayer & a_Player, NIBBLETYPE a_Meta);
```

Comment:
Looks like the 32-bit arm version does keep the arguments on the stack but arm64 does the same as x64. Though in this case it's mostly called with a temporary in which case the reference is strictly more work regardless of platform.